Assets/Scripts/Runtime/ItemPickCounter.cs

```diff
@@ -0,0 +1,18 @@
+﻿
+namespace Assets.Scripts.Runtime
+{
+    [System.Serializable]
+    public class ItemPickCounter
+    {
+        public BonusItemType _id;
+        public int _amount;
+
+        public string GetID { get { return _id.ToString(); } }
+        public string GetCurrentAmountAsString { get { return _amount.ToString(); } }
```

Comment:
jak robisz properte to bez `Get`
`GetAbc()` to funkcje i jak widzę Get to szukam nawiasów...